app/components/new-project-modal/template.hbs

```diff
@@ -80,12 +80,12 @@
             </div>
         {{/modal.body}}
         {{#modal.footer}}
-            <button onclick={{action closeModal}} class="btn btn-default">{{t 'general.cancel'}}</button>
+            <button onclick={{action closeModal false}} class="btn btn-default">{{t 'general.cancel'}}</button>
             <button onclick={{action create nodeTitle description searchSelected.id}} class="btn btn-success {{unless nodeTitle.length 'disabled'}}">{{t 'general.create'}}</button>
         {{/modal.footer}}
     {{else}}
         {{#modal.body}}
-            <button data-dismiss="modal" aria-label="Close" class="close" onclick={{action closeModal}}>
+            <button data-dismiss="modal" aria-label="Close" class="close" onclick={{action closeModal true}}>
```

Comment:
Might make more sense to use `modal.header`, for consistency with above
```hbs
{{#modal.header onClose=(action closeModal true)}}
    {{! empty? }}
{{/modal.header}}
```

Reply:
Modal.header adds a line implying that there should be a heading. This is done this way to match the look without the header

Reply:
also semantically there is no header in the post-success modal

Reply:
Makes sense 👍